Gravity/Gravity.Test.Integration/RSAPI_IntegrationTest.cs

```diff
@@ -0,0 +1,395 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using kCura.Relativity.Client;
+using NUnit.Framework;
+using Relativity.API;
+using Relativity.Test.Helpers;
+using Relativity.Test.Helpers.ServiceFactory.Extentions;
+using Relativity.Test.Helpers.SharedTestHelpers;
+using Relativity.Test.Helpers.WorkspaceHelpers;
+using Gravity.Test;
+using System.Configuration;
+using System.Diagnostics.Eventing.Reader;
+using Gravity.Base;
+using Gravity.Test.TestClasses;
+using kCura.Relativity.Client.DTOs;
+using System.Reflection;
+using Gravity.Test.Helpers;
+
+namespace Gravity.Test.Integration
+{
+    [TestFixture]
+    public class RSAPI_IntegrationTest
+    {
+        #region Variables
+
+        //this is set to true and should skip the creation of the database, installation of the testing application, and the deletion of the workspace.
+        //must point _workspaceId = valid workspace with application already installed in the Setup
+        private bool _debug = Convert.ToBoolean(ConfigurationManager.AppSettings["Debug"]);
```

Comment:
Ensuring this conversation does not get lost without further discussion: https://github.com/tsdservices/Gravity/pull/48#discussion_r174294950